lib/web_ui/lib/src/engine/semantics/link.dart

```diff
@@ -18,9 +18,10 @@ class Link extends PrimaryRoleManager {
   @override
   DomElement createElement() {
     final DomElement element = domDocument.createElement('a');
-    // TODO(mdebbar): Fill in the real link once the framework sends entire uri.
-    // https://github.com/flutter/flutter/issues/150263.
     element.style.display = 'block';
+    if (semanticsObject.hasLinkUri) {
```

Comment:
Can the link change? If yes, this code should probably go into an override of the `update` method and it should also check for `isLinkUriDirty`.

Reply:
Good catch!

Reply:
Done